ui/src/containers/SampleGridTableContainer.jsx

```diff
@@ -966,7 +973,7 @@ export default function SampleGridTableContainer(props) {
     const isFLEX = type.includes('FLEX');
     const isMOCK = type.includes('Mock');
 
-    if (isFLEX) {
+    if (isFLEX && !isSingleCell) {
```

Comment:
Just to match the order of other conditions:

```suggestion
    if (!isSingleCell && isFLEX) {
```

Reply:
Is the Flex sample changer ever single cell (with what we want to do ?)

Reply:
we have this : const isSingleCellAndNotFlex = isSingleCell && !type.includes('FLEX');
this could also  be 
const isFlexAndIsSingleCell =  type.includes('FLEX') && isSingleCell ;

i found it simpler to check  isSingleCellAndNotFlex

Reply:
Ah, I was thinking is it at all needed but lets not make things more complicated at the moment. I mean that we could simply say that all sample changers have at least one cell and that would simplify the logic a bit but lets keep it like this for the time being.